src/dash-table/DataTable.js

```diff
@@ -537,6 +540,48 @@ export const propTypes = {
         })).isRequired
     })),
 
+    column_static_tooltip: PropTypes.objectOf(PropTypes.shape({
```

Comment:
`objectOf` defines the shape of each property but not the name of those properties, this is exactly equivalent to the TS typing

```
interface IMyInterface {
    [key: string]: IMyInterfaceProp;
}
```

Reply:
I was unaware before that (a) PropTypes supported this, (b) Dash supported this (we support a pretty comprehensive subset of PropTypes but not *quite* all of it afaik) -- there's some debt in other types (especially conditional dropdowns) that would benefit from type improvements / better React props validation